packages/lucide-preact/rollup.config.mjs

```diff
@@ -22,32 +22,26 @@ const bundles = [
     format: 'cjs',
     inputs,
     outputDir,
-    aliasesSupport: true
   },
   {
     format: 'esm',
     inputs,
     outputDir,
     preserveModules: true,
-    aliasesSupport: true
   },
 ];
 
 const configs = bundles
-  .map(({ inputs, outputDir, format, minify, preserveModules, aliasesSupport }) =>
+  .map(({ inputs, outputDir, format, minify, preserveModules }) =>
     inputs.map(input => ({
       input,
       plugins: [
-        ...(
-          !aliasesSupport ? [
-            replace({
-              "export * from './aliases';": '',
-              "export * as icons from './icons';": '',
-              delimiters: ['', ''],
-              preventAssignment: false,
-            }),
-          ] : []
-        ),
+        replace({
+          "export * from './aliases';": '',
+          "export * as icons from './icons';": '',
+          delimiters: ['', ''],
+          preventAssignment: false,
+        }),
         ...plugins(pkg, minify)
       ],
```

Comment:
This can be removed entirely.
Now it remove the exports, that is not what we want.

So the correct config should be:
```js
plugins: plugins(pkg, minify)
```

Reply:
Oh dear, totally my bad. 🤦‍♂️
I've fixed them all.